webui/react/.storybook/preview.js

```diff
@@ -2,7 +2,69 @@ import 'shared/styles/index.scss';
 import 'shared/styles/storybook.scss';
 import 'shared/prototypes';
 
-import ThemeDecorator from "storybook/ThemeDecorator"
+import StoreProvider, { StoreAction, useStoreDispatch } from 'contexts/Store';
+import { ThemeOptions } from 'components/ThemeToggle';
+import useTheme from 'hooks/useTheme';
+import { BrowserRouter } from 'react-router-dom';
+import { useEffect } from 'react';
 
-export const decorators = [ ThemeDecorator ];
+export const globalTypes = {
+  // https://storybook.js.org/addons/@storybook/addon-toolbars
+  theme: {
+    name: 'Theme',
+    description: 'Global theme for components',
+    defaultValue: ThemeOptions.light.displayName,
```

Comment:
nit default to the system? if that's what we do in core. or maybe it's better to keep it at "light" always 🤔

Reply:
I'm fine with either way. @hkang1 any preference?